src/main/java/com/traveltime/sdk/dto/responses/geocoding/Properties.java

```diff
@@ -0,0 +1,45 @@
+package com.traveltime.sdk.dto.responses.geocoding;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+@Value
+@Builder
+@Jacksonized
+@AllArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Properties {
+
+    @NonNull
+    String name;
+
+    @NonNull
+    String label;
+
+    Double score;
+    String category;
+    String type;
+    String houseNumber;
+    String street;
+    String region;
+    String regionCode;
+    String neighbourhood;
+    String county;
+    String macroregion;
+    String city;
+    String town;
+    String district;
+    String country;
+    String countryCode;
+    String localAdmin;
+    String continent;
+    String postcode;
+
+    com.traveltime.sdk.dto.responses.mapinfo.Feature features;
```

Comment:
should it be here?

Reply:
The 'features' field of type mapinfo.Feature is part of the geocoding properties, however I removed the fully qualified path as it was not needed anymore.